### DISCUSSION TITLE
can't access mailing list linked from contribute page

### DISCUSSION THREAD

**Discussion by dorianmariecom** [rubygems/rubygems.org#2855] [General]
https://groups.google.com/access-error?continue=https://groups.google.com/g/gemcutter

<img width="1014" alt="Screenshot 2021-11-09 at 11 31 11" src="https://user-images.githubusercontent.com/58794487/140908345-0581f1c8-09b5-4b58-8bdf-0f81cd0715d6.png">

is this expected?

it's linked from https://guides.rubygems.org/contributing/

maybe this discussions thing is the new mailing list?

**Comment by simi** [comment]
That page is AFAIK full of obsolete information. Currently most of the discussions are happening on rubygems unified repo, in rfcs repo or on slack channel (http://slack.bundler.io/).

_btw. guides has custom repo - https://github.com/rubygems/guides_

**Reply by dorianmariecom** [reply]
thanks, see you on the slack

**Comment by deivid-rodriguez** [comment]
Yeah, can you create a PR to our guides repo to remove the link to the mailing list?

**Reply by dorianmariecom** [reply]
https://github.com/rubygems/guides/pull/298/files :)
